source/apiVolontaria/volunteer/tests/tests_admin_Participation.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth.models import User
+from rest_framework.test import APITransactionTestCase
+
+from django.contrib.admin.sites import AdminSite
+from django.test import Client
+
+from apiVolontaria.factories import UserFactory
+from ..factories import ParticipationFactory
+from ..admin import ParticipationAdmin
+from ..models import Participation
+
+
+class ParticipationTests(APITransactionTestCase):
+
+    def setUp(self):
+        self.participation = ParticipationFactory()
+        self.site = AdminSite()
+
+    def create_user(self):
+        self.username = "test_admin"
+        self.password = User.objects.make_random_password()
+        user, created = User.objects.get_or_create(username=self.username)
+        user.set_password(self.password)
+        user.is_staff = True
+        user.is_superuser = True
+        user.is_active = True
+        user.save()
+        self.user = user
```

Comment:
Creation of the user will be done for each test of this class, it could be better to do it in the `setUp` function :)

Reply:
This comment is always valid

Reply:
User now in setUp